docs/reference/contributing/guidelines/workflow.md

```diff
@@ -1,42 +1,60 @@
 ### Workflow
 
-#### Contributions
+#### Gatekeepers
```

Comment:
I don't think it makes sense to go straight into Gatekeepers without any sort of introductory paragraph. Could we move this down?

Reply:
OK will add here or move it